src/test/diagnostics_test.ts

```diff
@@ -205,35 +215,37 @@ suite('DiagnosticGenerator', function() {
 
     assert.deepEqual(
         (await client.getNextDiagnostics('index.html')).map(d => d.code), []);
-    test('sends diagnostics around polymer.json validation', async() => {
```

Comment:
Also this test was never run, it was just totally ignored. Merge error I hope :X

Reply:
oh wow, good catch!